docs/GettingStartedDocs/install_oe_sdk-Simulation.md

```diff
@@ -0,0 +1,28 @@
+# Install the Open Enclave SDK (non-SGX Ubuntu 16.04/18.04)
+
+## Considerations
+The Open Enclave SDK can be installed on non-SGX Ubuntu 16.04 and 18.04 systems. It can also be used in simulation mode, although some features may not work. Specifically these features are not supported when running in simulation mode:
+- Enclave signing and measurement
+- Data Sealing
+- Attestation (both remote and local)
+- Signal handling (specifically: the oe_add_vectored_exception_handler and oe_remove_vectored_exception_handler APIs)
+
```

Comment:
How about rephrasing this line to just "Intel SGX exception handling" without mentioning any APIs?

Reply:
OK, I am now convinced that we should reference the APIs.